pkg/agent/run.go

```diff
@@ -85,7 +85,7 @@ func Run(cmd *cobra.Command, args []string) {
 
 		b, err = ioutil.ReadAll(file)
 
-		credentials, err = ParseCredentials(b)
+		credentials, err = client.ParseCredentials(b)
```

Comment:
Perhaps also validate credentials at this point?

Reply:
`ParseCredentials` does validation.

Reply:
But not `nil || credentials.UserID == "" || credentials.UserSecret == ""`?

Reply:
(my concern was that we appeared to be validating the credentials when creating the client instead - but perhaps a non issue)